tests/api_connexion/endpoints/test_event_log_endpoint.py

```diff
@@ -291,6 +291,38 @@ def test_should_filter_eventlogs_by_when(self, create_log_model, session):
             assert response.status_code == 200
             assert {eventlog["event"] for eventlog in response.json["event_logs"]} == expected_eventlogs
 
+    def test_should_filter_eventlogs_by_included_events(self, create_log_model, session):
```

Comment:
```suggestion
    def test_should_filter_eventlogs_by_included_events(self, create_log_model):
```

If so, we can also get rid of the session fixture.